athenahealth/appointments.go

```diff
@@ -94,12 +96,12 @@ type BookedAppointment struct {
 	AppointmentStatus          AppointmentStatus `json:"appointmentstatus"`
 	AppointmentType            string            `json:"appointmenttype"`
 	AppointmentTypeID          string            `json:"appointmenttypeid"`
-	CancelledBy                string            `json:"cancelledby"`
-	CancelledDatetime          string            `json:"cancelleddatetime"`
 	CancelReasonID             string            `json:"cancelreasonid"`
 	CancelReasonName           string            `json:"cancelreasonname"`
 	CancelReasonNoShow         bool              `json:"cancelreasonnoshow"`
 	CancelReasonSlotAvailable  bool              `json:"cancelreasonslotavailable"`
+	CancelledBy                string            `json:"cancelledby"`
```

Comment:
Just noticed that these are intended to be alphabetical so would be nice to leave as they were but not blocking